backend/orders/views.py

```diff
@@ -3,17 +3,39 @@
 from rest_framework import status
 from rest_framework.decorators import api_view
 from rest_framework.response import Response
-from orders.models import Order
-from orders.serializers import OrderSerializer
+from .models import Order
+from .serializers import OrderSerializer
 
 
 @api_view(['GET', 'POST'])
 def order_list(request, format=None):
     """
-    List all code orders, or create a new order.
+    List all code orders, filter orders, or create a new order.
     """
+    token = request.GET.get('token')
+    if token is None or token == '':
+        return Response(status=status.HTTP_400_BAD_REQUEST)
+    weightMin_query = request.GET.get('weightMin')
+    weightMax_query = request.GET.get('weightMax')
+    costMin_query = request.GET.get('costMin')
+    costMax_query = request.GET.get('costMax')
+    status_query = request.GET.get('status')
+    if weightMax_query == '' or weightMin_query == '' or costMax_query == '' \
+            or costMin_query == '' or status_query == '':
+        return Response(status=status.HTTP_400_BAD_REQUEST)
```

Comment:
But these fields are not required, we do not have pass them to get the result

Reply:
I meant here that user can't do requests such:
/orders/?token=123&costMin=&costMax=
It should be like 
/orders/?token=123&costMin=12&costMax=100
or
/orders/?token=123

Reply:
When i tried to do this '''/orders/?token=123''', I have got an error. I think that it is because "" is default value of the parameters

Reply:
And it think that `/orders/?token=123&costMin=&costMax=` have the same meaning that `/orders/?token=123`